soc/arm/nxp_kinetis/kwx/soc_kw4xz.c

```diff
@@ -69,6 +70,10 @@ static ALWAYS_INLINE void clock_init(void)
 #if DT_HAS_NODE(DT_NODELABEL(lpuart0))
 	CLOCK_SetLpuartClock(LPUART0SRC_OSCERCLK);
 #endif
+
+#if CONFIG_PWM_0 || CONFIG_PWM_1 || CONFIG_PWM_2
```

Comment:
use nodelabel here.

Reply:
I think it's the same discussion as above: why enable the clock (that draws current) to an IP that is not used, since the user doesn't need/want it